Client/Coordinators/Library/BookmarksCoordinator.swift

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Storage
+
+protocol BookmarksCoordinatorDelegate: AnyObject {
+    func start(from cell: BookmarksFolderCell)
```

Comment:
This doesn't have the same parameter as the one down here `cell` instead of `folderCell`
```suggestion
    func start(from folderCell: BookmarksFolderCell) {
```

Reply:
yes thank you i noticed right now, i'm going to modify them. strange that i didn't get error during compilation

Reply:
Yeah thought the same!